docs/contracts/libraries/Validation.sol/library.Validation.md

```diff
@@ -0,0 +1,289 @@
+function getDepositsInL(InputParams memory inputParams)
+    private
+    pure
+    returns (uint256 netDepositedXinL, uint256 netDepositedYinL);
+```
+
+### getBorrowedInL
+
+
+```solidity
+function getBorrowedInL(InputParams memory inputParams)
+    private
+    pure
+    returns (uint256 netBorrowedXinL, uint256 netBorrowedYinL);
+```
+
+### convertXToL
+
+The original math:
+L * activeLiquidityScaler = x / 2 * sqrt(p)
+previous equation:
+amountL = Math.mulDiv(amount, POW2_96, 2 * sqrtPriceX96Range, rounding);
+adding activeLiquidityScaler:
+amountL = (amount * POW2_96 / (2 * sqrtPriceX96Range)) / (activeLiquidityScaler / POW2_96);
+simplify to:
+(amount * POW2_96 * POW2_96) / (2 * sqrtPriceX96Range * activeLiquidityScaler)
+final equation:
+amountL = Math.mulDiv(Math.mulDiv(amount, POW2_96, sqrtPriceX96Range, rounding), POW2_96, 2 * activeLiquidityScaler, rounding);
+or more simplified (failed for some tests)
+amountL = Math.mulDiv(amount, POW2_96 * POW2_96, 2 * sqrtPriceX96Range * activeLiquidityScaler);
+
+
+```solidity
+function convertXToL(
+    uint256 amount,
+    uint256 sqrtPriceX96Range,
+    uint256 activeLiquidityScaler,
+    Math.Rounding rounding
+) private pure returns (uint256 amountL);
+```
+
+### convertYToL
+
+The simplified math: L = y * sqrt(p) / 2
+Math.mulDiv(amount, sqrtPriceX96Range, 2 * POW2_96, rounding);
+amountL = amount * sqrtPriceX96RangeScaled / 2 * POW2_96
+sqrtPriceX96RangeScaled = sqrtPriceX96Range / activeLiquidityScaler / POW2_96;
+simplify to:
+amount * sqrtPriceX96Range / activeLiquidityScaler / POW2_96 / 2 * POW2_96
+simplify to:
+(amount * sqrtPriceX96Range) / (activeLiquidityScaler * 2)
+final equation:
+amountL = Math.mulDiv(amount, sqrtPriceX96Range, 2 * activeLiquidityScaler, rounding);
+
+
+```solidity
+function convertYToL(
+    uint256 amount,
+    uint256 sqrtPriceX96Range,
+    uint256 activeLiquidityScaler,
+    Math.Rounding rounding
+) private pure returns (uint256 amountL);
+```
+
+### checkLtv
+
+
+```solidity
+function checkLtv(CheckLtvParams memory checkLtvParams, InputParams memory inputParams) private pure;
+```
+
+### increaseForSlippage
+
+Calculates the impact of buying debt in the dex with slippage simplified to only use liquidity and
+
+*Uses a few formulas to simplify to not need reserves to calculate the required collateral to buy the debt.
+```math
+phi = 1 - fee
+reserveIn = { L * sqrt(p) if Lx > Ly`
+{ L / sqrt(p) otherwise
+reserveOut = { L / sqrt(p) if Lx > Ly
+{ L * sqrt(p) otherwise
+in * phi = reserveOut * reserveIn / (reserveOut - out) - reserveIn
+in * phi = reserveIn * (reserveOut / (reserveOut - out) - 1)
+in * phi = reserveIn * (reserveOut - (reserveOut - out) / (reserveOut - out)
+in * phi = reserveIn * (out / (reserveOut - out))
+```
+if Lx > Ly:
+math ```
+inL * 2 * sqrt(p) * phi = L * sqrt(p) * (outL * 2 / sqrt(p) / (L / sqrt(p) - outL * 2 / sqrt(p)))
+note in and out * 2 due to being half of the liquidity which is both x and y
+inL * phi = L / (L - 2 * outL)
```

Comment:
Katex seems to be throwing an error here:
```
ParseError: KaTeX parse error: Expected '}', got 'EOF' at end of input: …rveOut - out))
```

<img width="1013" alt="validation-error" src="https://github.com/Ammalgam-Protocol/docs/assets/5172086/9adae44b-8bc6-4ad9-806e-97da5b2d276e">

Reply:
Corrected these issues. now it is rendering correctly. Opened PR in core repo with changes. Probably more could be done in this library to clean up docs, but LaTeX is sort of a bitch. 

   
![Screen Shot 2023-11-15 at 3 39 00 PM](https://github.com/Ammalgam-Protocol/docs/assets/26027325/69ec6602-0f82-43d2-9347-b8ff56d69c2d)